examples/nlp/dialogue_state_tracking/conf/sgdqa_config.yaml

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+pretrained_model: null # pretrained QAModel model from list_available_models()
+do_training: true # training mode, for testing change to false
+trainer:
+  gpus: 8 # number of GPUs (0 for CPU), or list of the GPUs to use e.g. [0, 1]
+  num_nodes: 1
+  max_epochs: 3
+  max_steps: null # precedence over max_epochs
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  gradient_clip_val: 1.0
+  amp_level: O1 # O1/O2 for mixed precision
+  precision: 16 # Should be set to 16 for O1 and O2 to enable the AMP.
+  accelerator: ddp
+  log_every_n_steps: 5  # Interval of logging.
+  val_check_interval: 1.0  # Set to 0.25 to check 4 times per epoch, or an int for number of iterations
+  resume_from_checkpoint: null # The path to a checkpoint file to continue the training, restores the whole state including the epoch, step, LR schedulers, apex, etc.
+  num_sanity_val_steps: 0 # number of steps to perform validation steps for sanity check the validation process before starting the training, setting to 0 disables it
+  checkpoint_callback: False  # Provided by exp_manager
+  logger: False  # Provided by exp_manager
+
+model:
+  nemo_path: null # filename to save the model and associated artifacts to .nemo file
+  tokenizer:
+      tokenizer_name: ${model.language_model.pretrained_model_name} # or sentencepiece
+      vocab_file: null # path to vocab file
+      tokenizer_model: null # only used if tokenizer is sentencepiece
+      special_tokens: null
+
+  language_model:
+    pretrained_model_name: bert-base-cased
+    lm_checkpoint: null
+    config_file: null # json file, precedence over config
+    config: null
+
+  encoder:
```

Comment:
Is it OK to drop the encoder level as it is just one config. You may rename it to encoder_dropout. But both should work.

Reply:
i would like to keep it, since for other models its more parameters for encoder